ansible/templates/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.3"
+
+services:
+
+  db:
+    image: postgres:12-alpine
+    restart: always
+    environment:
```

Comment:
Wouldn't this go away, and just be:
```
 env_file:
    - .env
```

Reply:
No, we still need to specify which env var is passed to which container, it doesnt do that automatically.

Reply:
Ah k.